README.md

```diff
@@ -1,40 +1,52 @@
 # LDS Scriptures
 
-LDS Scriptures port to native command line in Go.
+![](data/web/static/favicon.ico)
 
-LDS Scriptures is a set of tools for downloading, parsing, and reading the Gospel Library content from [The Church of Jesus Christ of Latter-day Saints](http://lds.org). 
+ldss is a set of tools for downloading, parsing, and reading the Gospel Library content from [The Church of Jesus Christ of Latter-day Saints](http://lds.org). 
```

Comment:
The copyright content is also displayed here throughout the site. There are two apps. This is probably the best link for reference to contact or understand origin.